jerry-core/jerry.c

```diff
@@ -591,6 +591,41 @@ jerry_get_array_index_value (jerry_object_t *array_obj_p, /**< array object */
 } /* jerry_get_array_index_value */
 
 /**
+ * Get length of an array object
+ *
+ * Note:
+ *      Returns 0, if the given parameter is not an array object or
+ *      cannot retrieve the length.
```

Comment:
I am sorry I wasn't clear. So:
if array -> return length, always
if not -> return 0

So the correct comment is:
Returns 0, if the given parameter is not an array object